utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -1523,7 +1524,15 @@ public static boolean isIpWithtInCidrRange(final String ipAddress, final String
 
         final boolean isInRange = subnetUtils.getInfo().isInRange(ipAddress);
 
-        return isInRange;
+        return !isInRange;
```

Comment:
why did you reverse this return value @kansal ? I missed it in the first itteration